File: String/Java/CountChar.java
java.util.*;

class CountChar{
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);
        
        // Taking input Line
        String inputLine = input.nextLine();
        
        int vowel = 0, consonents = 0, digits = 0, symbols = 0;
        
        for(char ch: inputLine.toCharArray()){
            if(Character.isLetter(ch)){
                char lower = Character.toLower(ch);
                if(lower == 'a' || lower == 'e' || lower == 'i' || lower == 'o' || lower == 'u') vowel++;
                else consonents++;
            } else if(Character.isDigit(ch)) digit++;
            else symbols++;
        }
        
        System.out.println("Vowels: " + vowels);
        System.out.println("Consonents: " + consonents);
        System.out.println("Digits: " + digits);
        System.out.println("Symbols: " + symbols);
    }
}

File: String/CountChar.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    string inputString;
    getline(cin, inputString);
    
    int vowels = 0, consonents = 0, digits = 0, symbols = 0;
    
    for(char ch: inputString){
        if(isalpha(ch)){
            char lower = tolower(ch);
            if(lower == 'a' || lower == 'e' || lower == 'i' || lower == 'o' || lower == 'u') vowels++;
            else consonents++;
        }else if(isdigit(ch)) digits++;
        else symbols++;
    }
    
    cout << "Vowels: " << vowels << endl;
    cout << "Consonents: " << consonents << endl;
    cout << "Digits: " << digits << endl;
    cout << "Symbols: " << symbols << endl;
    
    return 0;
}
